ros/src/twist_controller/dbw_node.py

```diff
@@ -134,7 +154,7 @@ def upd_dbw_enabled(self, msg):
 
     def log_init(self, log_path):
         log_handle = open(log_path,'w')
-        headers = ','.join(["Time", "Target speed", "Target yaw", "Current speed", "DBW status", "Throttle", "Brake", "Steering"])
+        headers = ','.join(["p_effort", "i_effort", "d_effort", "pid_throttle", "feedforward_throttle", "velocity_error", "DT", "decel_target", "latchBrake", "dbw_time", "target_linear_velocity", "target_angular_velocity","current_linear_velocity", "current_angular_velocity", "dbw_status", "throttle", "brake", "steering"])
```

Comment:
There are less headers than values printed in the file. The reason is that there are 2 sets of [p_effort, i_effort, d_effort] printed:
- the first one is either for throttle or brake based on the brakelatch parameter
- the second is for steering angle

We need to ensure we have the 3 sets of [p, i, d] always printed and adjust the headers ["PID throttle - P", "PID throttle - I…]